src/api/models/article.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable new-cap */
+export default (sequelize, DataTypes) => {
+  const Article = sequelize.define('Article', {
```

Comment:
You should have the same name of the table @Ramadhan0. It should be the same as for migrations